src/app/content/highlights/components/ConfirmationModal.tsx

```diff
@@ -1,28 +1,34 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
 import Button from '../../../components/Button';
 import Modal from '../../../components/Modal';
+import { modalPadding } from '../../../components/Modal/styles';
 
 interface Props {
   deny: () => void;
   confirm: () => void;
 }
 
 // tslint:disable-next-line:variable-name
-const ConfirmationModal = ({deny, confirm}: Props) => <Modal
-  onModalClose={deny}
-  heading='i18n:discard:heading'
-  subheading='i18n:discard:body'
-  footer={
-    <>
+const Footer = styled.div`
```

Comment:
define and export footer from `../../../components/Modal/styles'` with the flex and padding, you can extend it here with the justify content by doing 

```
const ConfirmationFooter = styled(Footer)`
  justify-content: space-between;
`;
```